subjects/PE-x86/VCExeSample/VCExeSample.c

```diff
@@ -10,9 +10,9 @@ int32 main(int32 argc, char * * argv)
 	return 0x00;
 }
 
-void test1(char * arg1, int32 arg2, char * arg3, real32 arg4)
+void test1(char * arg1, int32 arg2, char * arg3, Eq_10 arg4)
 {
-	printf("%s %d %s %f");
+	printf("%s %d %s %f", arg1, arg2, arg3, arg4);
 	return;
```

Comment:
We need to take into account that when calling a varargs function, real32's are converted to real64's. That would avoid creating the union type `Eq_10` above.

For now, we live with it, but I'd probably prefer seeing:
`printf("%s %d %s %f", arg1, arg2, arg3, (double) arg4);`